### ISSUE TITLE
Instructions for deconz and hue not clear

### ISSUE THREAD

**Issue by PrayerfulDrop** [mariusmotea/diyHue#467]
Hello,

I have a Raspbee Zigbee module on my RiPi3B+.  Both diyHUE and deconz are running correctly.  I have a Tradfri bulb connected to deconz and can control it without issues inside the http://<diyHUE_IP>:8080.  I am following the directions in the wiki to connect diyHUE and deconz to import the bulbs but nothing is working.  Help??

**Comment by mariusmotea** [comment]
Hi. Did you enable app pair in Deconz in order to allow hue emulator to register with deconz? How you check the lights, from internal webgui diyhue display just the groups and the lights inside the groups. Fresh detected lights are not displayed

**Comment by PrayerfulDrop** [comment]
That is the issue I am having.  I cannot get the two to work.

Here is what I am doing:
1. Navigate to http://10.0.2.152:8080
![image](https://user-images.githubusercontent.com/999665/49669234-5c5b7900-fa26-11e8-8f94-f793d1a6d467.png)
2. Click Authenticate app
3. Navigate to http://10.0.2.152/deconz
4. Click Save

This is based on the wiki for raspbee.

**Comment by mariusmotea** [comment]
Corect, but you check the lights with hue app or hue essentials?

**Comment by PrayerfulDrop** [comment]
On iOS or in the diyHue app?

**Comment by mariusmotea** [comment]
Check in ios hue app for the new lights. If are not there start the script in debug mode and put here the output to see whay is hapenning.

**Comment by PrayerfulDrop** [comment]
Should I see anything in the diyHue web page?

**Comment by mariusmotea** [comment]
No, you see only after you setup the lights and add them in the groups.

**Comment by cheesemarathon** [comment]
Is this issue fixed @PrayerfulDrop

**Comment by PrayerfulDrop** [comment]
Nope it isn’t. I am starting over from square 1. Something got hosed in diyHue and now the Hue app doesn’t see the bridge. Frustrating.

**Comment by cheesemarathon** [comment]
Ok, @PrayerfulDrop Sorry, your having issues still. I'm going to close this now. If you need more help could you open a new issue at our new repo [here](https://github.com/diyhue/diyHue), we moved! Thanks!
